practice-two/src/js/script.js

```diff
@@ -150,9 +153,6 @@ const isvalidateForm = () => {
  * 
  */
 const saveData = () => {
-  // Parse any JSON previously stored in allEntries
-  let existingEntries = JSON.parse(localStorage.getItem(UsersKey));
-
   // If existingEntries is null, array will be created
   if(existingEntries === null) {
```

Comment:
If you move load all user from LocalStorage outside. Why its condition still here?